pythonik/models/files/file.py

```diff
@@ -37,7 +37,7 @@ class File(BaseModel):
     checksum: Optional[str] = ""
     date_created: Optional[str] = ""
     date_modified: Optional[str] = ""
-    directory_path: Optional[str] = ""
+    directory_path: Optional[str] = None
```

Comment:
This Optional line should default to an empty string to match the type.

Reply:
If it's an empty string it'll still be falsey. But if it's referenced in a string way it won't throw errors.